src/main/resources/TotemCode/TotemSheet.xml

```diff
@@ -545,7 +545,7 @@ require(['jquery', 'colpick'], function($, Colpick) {
     <name>TotemCode.TotemSheet</name>
     <number>2</number>
     <className>XWiki.JavaScriptExtension</className>
-    <guid>31a5edfd-5305-444c-b817-fc604f35a422</guid>
```

Comment:
These changes should not be commited as there are not usefull. This is not a blocker so it can be commited as is but it would be better to avoid this next time.